jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/HandlerRegistry.java

```diff
@@ -72,7 +86,8 @@ static HandlerRegistry createDefault(MeterProvider meterProvider) {
             new ContainerConfigurationHandler(),
             new LongLockHandler(grouper),
             new ThreadCountHandler(),
-            new ClassesLoadedHandler());
+            new ClassesLoadedHandler(),
+            new OldGarbageCollectionHandler(garbageCollectors));
```

Comment:
These could be added in the switch statement as well. Instead of passing the set of `garbageCollectors` you could pass the name of the garbage collector as a constructor argument.

Reply:
This is a good idea. It also allows us to get rid of the set caching all the GC types. Done!